### DISCUSSION TITLE
Welcome to AzUnzipEverything Discussions!

### DISCUSSION THREAD

**Discussion by fboucher** [FBoucher/AzUnzipEverything#26] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dpiva** [comment]
Great work on the function app! I was able to deploy it and test it out on my Azure account with no issues. I do have a few questions. How do I use this in a logic app? I tried calling it and there were no triggers or actions that I could utilize to point it to a different storage account or destination container. Any insight on how I could make modifications to it so I can use it via a logic app? Also, how can I change the default storage account to a different one? Thank you!
